### PR TITLE
CL Release 18.08.2022

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

File: cl-base.obo [hunk 1]
```diff
@@ -21269,9 +21273,10 @@ name: lymph node marginal reticular cell
 def: "A fibroblastic reticular cell found in the lymph node subcapsular sinus." [https://orcid.org/0000-0001-7655-4833, PMID:21435450, PMID:22566825, PMID:25998961, PMID:28262465, PMID:29752062, PMID:34256989]
 synonym: "marginal reticular cell" BROAD []
 is_a: CL:0009102 ! lymph node fibroblastic reticular cell
-relationship: BFO:0000050 UBERON:0005463
+relationship: RO:0001025 UBERON:0005463
 property_value: http://purl.org/dc/elements/1.1/date 2022-03-21T15:59:56Z xsd:dateTime
 property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-2825-0621
+property_value: IAO:0000116 "The use of 'adjacent to' is due to subcapsular sinus being an anatomical space and hence a immaterial entity. For more information see https://github.com/obophenotype/cell-ontology/issues/1636" xsd:string
```
Comment (shawntanzk) [inline cl-base.obo:21279]: sorry, I forgot to remove this comment. I know that this is urgent so happy to push this through leaving it while i fix the comment in -edit if need be.

**Review by shawntanzk — APPROVED** [review]
other than the comment not being removed, looks good

**Comment by matentzn** [comment]
It's fine just remove in master and make a release now I think!

**Comment by shawntanzk** [comment]
I've already removed it master, can make a release later tonight, or if you are able to, happy for you make the release and I will review it tonight :) thanks

**Comment by matentzn** [comment]
Ok will do

**Comment by matentzn** [comment]
ready @shawntanzk

**Pull request merged**